### PR TITLE
Some style cleanups in Stage1.

### PR DESCRIPTION
**Pull request by farmdawgnation**
Some suggestions for #25. Probably some more things I'll fine when I get another chance to sit down and tinker with it, but I wanted to open early and provide some opportunity for feedback before I've invested a day in it. :grin:

### CODE REVIEW

File: stage1/Stage1.scala [hunk 1]
```diff
@@ -1,70 +1,81 @@
...
+
+import paths._
 
 object CheckAlive{
   def main(args: Array[String]): Unit = {
     System.exit(33)
   }
 }
+
+class Init(args: Array[String]) {
```
Comment (cvogt) [inline stage1/Stage1.scala]: We should probably start making things private[cbt] in order to reduce the exposed interface

File: stage1/Stage1.scala [hunk 2]
```diff
@@ -1,70 +1,81 @@
...
+  /**
+   * Raw parameters including their `-D` flag.
+  **/
```
Comment (cvogt) [inline stage1/Stage1.scala]: If it's a one liner, I generally prefer the more compact
```
/** Raw parameters including their `-D` flag. */
```

**Comment by cvogt** [comment]
This is great. LGTM

File: stage1/Stage1.scala (hunk 1 above)
Comment (farmdawgnation) [inline stage1/Stage1.scala]: Good idea.

File: stage1/Stage1.scala (hunk 2 above)
Comment (farmdawgnation) [inline stage1/Stage1.scala]: Ah, sure. That makes sense.

Comment (cvogt) [inline stage1/Stage1.scala]: actually, I take that back. let's not get into whitespace discussions. once there is scalariform, that's all unified anyways.

File: stage1/Stage1.scala (hunk 1 above)
Comment (cvogt) [inline stage1/Stage1.scala]: dropped it again in order to access it from the tests. maybe those should go into the same package. they test via processes not jvm anyways

**Pull request merged**